indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -307,22 +308,27 @@
    Values in log:
    * fv : Format Version
    * mg : MessaGe type
+   * ct : Concept Type (from Concept Id)
    * ci : Concept Id
    * ri : Revision Id (number)
-   * vm : Visibility Milliseconds
-   * ar : All Revisions (0 or 1)
+   * vm : Visibility Milliseconds (number)
+   * ar : All Revisions (0 for false or 1 for true)
 
    NOTE: Changes to this text may break reports."
   [concept-id revision-id milliseconds all-revisions-index?]
-  (let [all-value (if all-revisions-index? 1 0)]
-    (format (str "{\"fv\": 1 \"mg\": \"index-vis\", "
-                 "\"ci\": \"%s\", "
-                 "\"ri\": \"%s\", "
-                 "\"vm\": \"%d\", "
-                 "\"ar\": %d}")
+  (let [all-value (if all-revisions-index? 1 0)
+        concept-type (re-find (re-pattern "^[A-Z]+") concept-id)]
+    (format (str "{\"fv\":1,"
+                 "\"mg\":\"index-vis\","
```

Comment:
fv and mg are hard coded, is there any reason to keep these? I don't know what value they provide. 

also, how many bytes do we gain by making this a bit more human-readable, i.e. not using abbreviations?

Reply:
yes, as I write new logs, I will use a different mg value so that you can find one or the other. As for fv, this is the "format version". If I want to change the format radically I can change this value and then maybe you can query that and know what to do?